### PR TITLE
PEAR-1601+1602: Fix facet sort panel a11y and same id issue for live region

### PR DESCRIPTION
**Pull request by paribartandhakal**
## Description
Changes include:
1. correct use of icons
2. correct use of initial sorting direction
3. aria live just announces once after the sorting has been done

## Checklist
- [N/A] Added proper unit tests
- [N/A] Left proper TODO messages for any remaining tasks
- [N/A] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

## Screenshots/Screen Recordings (if Appropriate)
https://github.com/NCI-GDC/gdc-frontend-framework/assets/101295912/43b0fa89-4dd0-4af2-9c4b-660cb5d0f927

### CODE REVIEW

**Comment by cemile-ctds** [comment]
Tested with VoiceOver and it works fine on my side.

File: packages/portal-proto/src/features/facets/FacetSortPanel.tsx [hunk 1]
```diff
@@ -4,51 +4,68 @@ import {
...
+  setSort: (sort: SortType) => void;
+  field: string;
+}
+
+const sortTypeToAriaDescription = (
+  sortTypeAndDirection: SortType,
+  valueLabel: string,
+  field: string,
+) => {
+  console.log({ sortTypeAndDirection });
```
Comment (wteouchicago) [inline packages/portal-proto/src/features/facets/FacetSortPanel.tsx:22]: log?

**Comment by wteouchicago** [comment]
looks good to me. didn't spot anything unusual default-sort-wise or icon-direction-wise. 👍

File: packages/portal-proto/src/features/facets/FacetSortPanel.tsx (hunk 1 above)
Comment (paribartandhakal) [reply packages/portal-proto/src/features/facets/FacetSortPanel.tsx]: 🥇

File: packages/portal-proto/src/features/facets/FacetSortPanel.tsx [hunk 2]
```diff
@@ -74,22 +96,32 @@ const FacetSortPanel: React.FC<FacetSortPanelProps> = ({
...
         {valueLabel}
       </Button>
-      <span className="Offscreen">
-        <span id="liveRegion" aria-live="polite">
-          {sortTypeToAriaDescription(sortType, valueLabel)}
-        </span>
-      </span>
+
+      <span
+        id={`${field}-liveRegion`}
```
Comment (ocshawn) [inline packages/portal-proto/src/features/facets/FacetSortPanel.tsx:120]: is this Id needed? 

also this should not cause a problem but next time look into using react useId if you need to use an ID

**Review by ocshawn — APPROVED** [review]
looks good

**Review by craigrbarnes — APPROVED** [review]
Changes look fine to me

**Pull request merged**